packages/react-vapor/docs/src/styles/Menu.tsx

```diff
@@ -90,8 +90,6 @@ const Navigation: React.FunctionComponent = () => (
             <NavigationLink href="/site" name="Site" />
         </NavigationSection>
         <NavigationSection title="Icons" baseUrl="/icons" svgName="noContent">
-            <NavigationLink href="/modifiers" name="Icon modifiers" />
-            <NavigationLink href="/fillers" name="Icon fillers" />

```

Comment:
You deleted the navigation entries, but you could also remove the associated example files.